README.md

```diff
@@ -28,18 +28,18 @@ This library simplifies the process of writing a profile which can be converted
 Using [WorkflowBuilder from `workflow.py`](pegasus_wrapper/workflow.py) develop a function to generate a `Workflow.dax`.
 See [example_workflow](pegasus_wrapper/scripts/example_workflow_builder.py) for an extremely simple workflow which we will use to demonstrate the process.
 To see the example workflow add a `root.params` file to the parameters directory with the following:
-*Note the Directory should be in your $Home and not a NFS like /nas/gaia/ as the submission will fail for an NFS reason*
```

Comment:
Have you actually gotten this to work on `/nas/gaia/`? I know we discussed that @spigo900 was able to run this no problem on `/nas/minlp` but `/nas/gaia` might be set up differently.

Reply:
I believe I got my sample workflow to run from a directory in `/nas/gaia`